### ISSUE TITLE
Defined structure for articles?

### ISSUE THREAD

**Issue by TheBeege** [TheBeege/SciNet#11]
After reading the Vox article, I'm inclined to consider the idea of proposing a defined structure for articles. Out of this issue, I'd like a simple yes/no decision on whether we're interested in proposing structure. If we decide yes, we can discuss the different potential structures in other issues.

The article sites that poorly design studies and poorly written articles are causing unnecessary obfuscation. If we require authors to follow certain templates, especially those requiring reproduction steps, we can alleviate the issue. I imagine it may also reduce some of the writing effort for authors. My guess is that filling in the blanks is easier than free-form. I'd still want to leave free-form for areas like analysis.

Thoughts?

**Comment by DesireeVanHaute** [comment]
Yes. We should have a structure. A uniformed structure helps readers identify where they can find the information they need to analyze the article.

**Comment by anthonyselkowitz** [comment]
I concur, F1000 has a broad structure of what they accept in terms of article content(such as sections and formatting). This would be something to implement in our system as well.
